doc/node.1

```diff
@@ -219,5 +231,5 @@ Mailing list:
 IRC (general questions):
 .ur "chat.freenode.net #node.js"
```

Comment:
Nit: Given that `#node.js` is not *yet* considered an official resource managed by the project, a note indicating it's unofficial status might be worthwhile.